spec/ruby/language/predefined_spec.rb

```diff
@@ -1314,6 +1314,7 @@ def obj.foo2; yield; end
 
     it "returns what will be loaded without actual loading, .so file" do
       require 'rbconfig'
+      skip "static linked exts have no extension" if RbConfig::CONFIG["EXTSTATIC"] == "static"
```

Comment:
"have no extension" sounds inaccurate.
static-linked-ext doesn't prohibit extension libraries, but just make standard extensions to be installed linked statically.
In fact, extensions for tests are still dynamically loaded.

Reply:
"have no mandatory dynamically loadable extension" describes that situation precisely?